src/main/steps/common/common.content.ts

```diff
@@ -210,6 +210,11 @@ export const generatePageContent = ({
     userCase &&
     [State.Draft, State.AwaitingApplicant1Response, State.AwaitingApplicant2Response].includes(userCase.state!);
   const isClarificationAmendableState = userCase && userCase.state === State.AwaitingClarification;
+  const isFinalOrderState =
```

Comment:
not sure if this should be here if its only being used in 1 place, tho i guess the same can be said to the above variables.